tests/model_selection/test_wrappers.py

```diff
@@ -48,14 +54,30 @@ def _keras_build_fn(lr=0.01):
     return model
 
 
-def test_keras():
+@gen_cluster(client=True)
+def test_keras(c, s, a, b):
+#  def test_keras():
+    pytest.importorskip("tensorflow")
+    pytest.importorskip("scikeras")
+
     X, y = mnist()
     assert X.ndim == 2 and X.shape[-1] == 784
     assert y.ndim == 1 and len(X) == len(y)
+    assert isinstance(X, np.ndarray) and isinstance(y, np.ndarray)
 
     model = KerasClassifier(build_fn=_keras_build_fn, lr=0.1)
     params = {"lr": loguniform(1e-3, 1e-1)}
 
+    with pytest.warns(DataConversionWarning):
```

Comment:
Can you post this warning and a bit about what's happening?

Reply:
I think this is happening because SciKeras plays with the shapes: https://github.com/adriangb/scikeras/issues/19#issuecomment-658549923

Reply:
Yep. Keras and ScikitLearn don't always agree on the shape of things (ex: `(n_points, )` vs `(n_points, 1)`). SciKeras *tries* to convert back and forth to make everyone happy, but I'm not sure that it can make everyone happy and avoid modifying the user's data 100% of the time. I opened an issue to try and improve this behavior (and hopefully reduce the occurrence of that warning): https://github.com/adriangb/scikeras/issues/20